setup.py

```diff
@@ -118,11 +118,16 @@ def has_ext_modules(self):
 with open(os.path.join(rootpath, "setup.py"), "w") as f:
   f.write(content.format(version, project))
 
+datapath = None
 if '--data' in sys.argv:
   data_idx = sys.argv.index('--data')
   datapath = sys.argv[data_idx + 1]
   sys.argv.remove('--data')
   sys.argv.pop(data_idx)
+else:
+  datapath = os.environ.get('TFIO_DATAPATH')
```

Comment:
@yongtang I added this as an alternative to `--data` option for when we might run `setup.py` manually and have the env var set. I didn't change the CI script to use this because the `--data` seems fine.

Reply:
@BryanCutler The change in setup.py is not needed. This file is only used for packaging the whl file. In the process of packaging, it always needs a source file and the .so file. At this stage, .so file is always in a separate directory than source (bazel). This setup.py file is also not part of the whl itself so it will not impact the behavior of `__init__.py`.

Reply:
I added this as a convenience if using the Docker image from #104. It sets the env var for the container, so if I want to build a wheel for my own testing I can use the command `python setup.py bdist_wheel`. It's just to make it slightly easier for the developer and the `--data` arg still takes precedence here. Would you still like me to remove it?

Reply:
OK. I think we could leave it alone.